book.rb

```diff
@@ -0,0 +1,12 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
```

Comment:
- [ ] Kindly make sure that a `@rentals` variable is present in the `Book` class so you can save rentals